x-pack/plugins/alerting/server/lib/snooze/is_snooze_active.test.ts

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import { RRule } from 'rrule';
+import sinon from 'sinon';
+import { RRuleRecord } from '../../types';
+import { isSnoozeActive } from './is_snooze_active';
+
+let fakeTimer: sinon.SinonFakeTimers;
+
+describe('isSnoozeExpired', () => {
+  afterAll(() => fakeTimer.restore());
+
+  test('snooze is NOT active byweekday', () => {
+    // Set the current time as:
+    //   - Feb 27 2023 08:15:00 GMT+0000 - Monday
+    fakeTimer = sinon.useFakeTimers(new Date('2023-02-27T08:15:00.000Z'));
```

Comment:
This fails if you change the time to `2023-02-27T06:15:00.000Z`.

Reply:
Good catch! Turns out I forgot to convert the `now` date that we're using to compare against the lastOccurence. Fixing this fixed another test too: https://github.com/elastic/kibana/pull/153972/commits/d757a43980364d95bfda6c3d8ab0daabe5be9538